nucleus/admin/rest/rest-service/src/main/java/org/glassfish/admin/rest/resources/CollectionLeafResource.java

```diff
@@ -246,6 +255,23 @@ protected ActionReportResult buildActionReportResult() {
         ResourceUtil.addMethodMetaData(ar, mmd);
         return new ActionReportResult(ar, optionsResult);
     }
+    
+    private Map<String, String> optionToMap(JvmOption option){
+        Map baseMap = new HashMap<>();
+        if (option.minVersion.isPresent()) {
+            baseMap.put(MIN_VERSION, option.minVersion.get());
+        } else {
+            baseMap.put(MIN_VERSION, "");
```

Comment:
If both paths have the left hand side in common I'd use `? option.minVersion.get()) : ""`